code/modules/surgery/transorbital_lobotomy.dm

```diff
@@ -0,0 +1,63 @@
+/datum/surgery/transorbital_lobotomy
+	name = "Transorbital Lobotomy"
+	steps = list(/datum/surgery_step/pick, /datum/surgery_step/drive, /datum/surgery_step/tlobotomize, /datum/surgery_step/close)
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_PRECISE_EYES)
+
+/datum/surgery/transorbital_lobotomy/can_start(mob/user, mob/living/carbon/target)
+	. = ..()
+	if(!.)
+		return
+	return target.getorganslot(ORGAN_SLOT_BRAIN) ? TRUE : FALSE
+
+
+/datum/surgery_step/tlobotomize
+	name = "perform transorbital lobotomy"
+	implements = list(TOOL_ORBITOCLAST = 75, /obj/item/melee/transforming/energy/sword = 50, TOOL_SCREWDRIVER = 15, /obj/item/stack/rods = 10)
+	time = 7.5 SECONDS
```

Comment:
another downside to removing normal lobotomies: you can't speed up any of the steps (aside from the last one) of a transorbital lobotomy using alien and/or advanced tools, because there aren't any advanced or alien equivalents for orbitoclasts and hammers.